internal/attacktechniques/k8s/privilege-escalation/hostpath-volume/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.7.1"
+    }
+  }
+}
+
+variable "kubeconfig" {
+  type        = string
+  description = "Path to KubeConfig"
+  default     = "~/.kube/config"
+}
+
+# Sourcing kubernetes credentials
+# 1. kubeconfig terraform variable
+# 2. ~/.kube/config
+# 3. Kubernetes in-cluster config
+provider "kubernetes" {
+  config_path = fileexists(pathexpand(var.kubeconfig)) ? pathexpand(var.kubeconfig) : null
+}
+
+resource "kubernetes_namespace" "namespace" {
```

Comment:
I'm concerned that this will clash when using multiple techniques at once. We probably need one namespace per TTP to avoid conflicts, I'll look into naming it more specifically

- [ ] Rename K8s namespace to avoid conflicts